src/core/io/ibprof_xml.c

```diff
@@ -161,10 +163,10 @@ static const char *_ibprof_hash_format_xml(int module, const char* call_name, co
 	ret = sys_vsnprintf(stat_buffer,
 		sizeof(buffer),
 		XML("count", "%ld") \
-		XML("total_ms", "%.4f") \
-		XML("avg_ms", "%.4f") \
-		XML("max_ms", "%.4f") \
-		XML("min_ms", "%.4f") \
+		XML("total", "%.4f") \
+		XML("avg", "%.4f") \
+		XML("max", "%.4f") \
+		XML("min", "%.4f") \
 		XML("fail", "%ld"),
```

Comment:
@liranoz12 It would be useful to provide user friendly information about (time period as s, ms, us).